### ISSUE TITLE
Duplicated RFC in Profile?

### ISSUE THREAD

**Issue by d3e3e3** [ietf-tools/datatracker#7886]
### Describe the issue

I came across a Datatracker profile for a person that lists the same RFC twice! In particular the profile for Margaret Cullen (https://datatracker.ietf.org/person/mrcullen42@gmail.com) lists RFC 8636 twice. Both are included in the count which claims she has 12 RFCs even though there are only 11 different RFCs list but 12 lines due to the duplicated RFC... In case there is something transient about this, I have printed that web page and attached the resulting PDF.
[Profile for Margaret Cullen.pdf](https://github.com/user-attachments/files/16839376/Profile.for.Margaret.Cullen.pdf)

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by richsalz** [comment]
Yes, it seems transient -- the current version shows 11 RFCs and 8636 is only done once. I wonder if work on #6957 or other work fixed it?  I suggest closing this, and re-open if it shows up again.

**Comment by hallambaker** [comment]
Has not returned since.

Suspect the cause is probably not specific to that author. Probably something that happens when things are being read when something else is occurring.

**Comment by rjsparks** [comment]
Closing as OBE. (I suspect that rich is correct and that at the time this was reported, we were still making rfcs first-class objects in the datatracker and there were still bugs being removed).
